src/Table/Table.js

```diff
@@ -33,12 +34,21 @@ function Table(){
         <tbody>
             {
                 response.map((item,i)=>{
-                    return (<tr>
+                    if (i%2==0){
```

Comment:
use the css odd even styling

something like this: tr:nth-child(even) {background-color: #f2f2f2;}
examples: https://www.w3schools.com/css/tryit.asp?filename=trycss_table_striped